users/api/http/responses.go

```diff
@@ -18,6 +18,15 @@ var _ mainflux.Response = (*tokenRes)(nil)
 type tokenRes struct {
 	Token string `json:"token,omitempty"`
 }
+type identityRes struct {
+	Email    string                 `json:"email"`
+	Metadata map[string]interface{} `json:"metadata,omitempty"`
+}
+
+type userInfo struct {
+	Email    string                 `json:"email"`
+	Metadata map[string]interface{} `json:"metadata,omitempty"`
+}
```

Comment:
These two are the same. Also, please add `code`, `headers` and `empty` methods to the one that you leave.